contents/docs/web-analytics/getting-started.mdx

```diff
@@ -0,0 +1,204 @@
+
+#### 📋 Use case: Tracking visitors from a sponsored video
+
+One example of how PostHog uses these features was a recent sponsored video on YouTube. We sponsored a [Fireship](https://www.youtube.com/c/Fireship) video, and we can see how it worked below.
+
+For the YouTube video, there's a link in the description for people that are interested in seeing what PostHog is, and it looks like this:
+
+`https://posthog.com/fireship`
+
+That URL gets transformed into the following UTM paramaters:
+
+`https://posthog.com/?utm_source=fireship&utm_campaign=fireship`
+
+Then, we can see how many visitors came from that video by using the "UTM source" option on our web analytics dashboard:
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/utm_source_light_21fa9c00fe.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/utm_source_dark_eada0f8178.png"
+  alt="UTM sources"
+  classes="rounded"
+/>
+
+### Demographic information
+
+Based on how users are accessing your website, web analytics also provides additional demographic metrics that can help answer the following:
+
+- What kinds of devices are visitors using? Laptops? Tablets? Phones?
+- Which browsers and operating systems are most popular?
+- Where are users located geographically? Which countries? Which cities?
+- What are the most popular languages for visitors?
+
+### Goals
+
+The goals section of the web analytics dashboard is all about tracking conversion goals.
+
+> **💡 Concept:** What is a conversion? A conversion is a key event or action you want users to do. If a user converted, it means they not only visited your website, but also performed some key action like signing up or subscribing.
+
+You can click the **Add conversion goal** button at the top of the web analytics dashboard, and check out our detailed guide on how to set up [conversion goals](/docs/web-analytics/conversion-goals).
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_04_at_12_29_08_3bba14e7e9.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_04_at_12_28_25_689e6c893d.png"
+  alt="Conversion goal"
+  classes="rounded"
+/>
+
+## Further reading
+
+This post is meant as a gentle introduction to the big picture concepts for web analytics, but there's a lot more depth available. Check out the web analytics [dashboard](/docs/web-analytics/dashboard) docs for more.
+
+And while web analytics is good for tracking high-level website metrics, product analytics is good for diving deep into how specific users behave. For more on the distinction between the two, check out our post on [web vs. product analytics](/docs/web-analytics/web-vs-product-analytics).
```

Comment:
```suggestion
```

I think we can lose this. This is more an element we're more likely to use in a tutorial than a docs page.